application/helpers/expressions/em_manager_helper.php

```diff
@@ -9800,6 +9752,81 @@ public static function htmlSpecialCharsUserValue($string)
         $string = str_replace(["{","}"], ["&#123;","&#125;"], $string);
         return $string;
     }
+
+    /**
+     * Save the starting values to the database
+     */
+    public static function SaveStartingValues()
+    {
+        // Init
+        $LEM =& LimeExpressionManager::singleton();
```

Comment:
`=&` is PHP4 legacy, better to just use `=` these days.

Reply:
Sorry, I know you just copied these lines. >< Now you have to pay the price! :laughing:

Reply:
My opinion : keep same method in big file like this. Except if really rewrite , no ?

Reply:
@olleharstedt Do you think I should change it only here or everywhere on this file?
I would just do nothing on this case.

Reply:
Bwah. Either only here or do nothing. We can create a new dev issue: "Remove PHP4 idioms".

Reply:
Sorry. I say nothing this time and vote for the new issue.